app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,44 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name')
+    const postTitle = this.postTitleMarkup;
+
+    // need this data !!
+    const isOwnWall = firstReplierName === '// check model.uri for username? in Message Wall uri';
```

Comment:
You'll want to reproduce this function here I think https://github.com/Wikia/unified-platform/blob/master/extensions/fandom/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotificationsTextFormatter.js#L192-L199, using the `model.get('uri')` as the parameter. HOWEVER! This method is broken for some usernames containing underscores, which is what CAKE-5920 is attempting to fix, but that hasn't been deployed yet. Newer notifications (after CAKE-5920 is deployed) will explicitly have the wall owner's username as a field in the notification model.

Reply:
Moved that function to `utils/messagewall.js`.